include/swift/AST/DiagnosticsSema.def

```diff
@@ -1314,8 +1314,8 @@ NOTE(inherited_protocol_does_not_conform,none,
      "type %0 does not conform to inherited protocol %1", (Type, Type))
 NOTE(no_witnesses,none,
      "protocol requires %select{initializer %1|function %1|property %1|"
-     "subscript}0 with type %2; do you want to add a stub?",
-     (RequirementKind, DeclName, Type))
+     "subscript}0 with type %2%select{|; do you want to add a stub?}3",
```

Comment:
While you're in here, maybe you can rearrange this so that `%select{initializer %1...}` is all on the same line? It might be helpful for readability, or if anyone ever wants to search using regex :)